transitions/utils.js

```diff
@@ -56,5 +56,18 @@ module.exports = {
       doc.patient_id = patientId;
       callback();
     }).catch(callback);
+  },
+  extractLineage: contact => {
```

Comment:
yes it is! 
They should also be called with matching names, e.g. `hydrateDoc` and `dehydrateDoc`